<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Marche</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="leftlayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="addHostBtn">
           <property name="text">
            <string>Add host</string>
           </property>
           <property name="icon">
            <iconset resource="../res/marche-gui.qrc">
             <normaloff>:/marche/server.png</normaloff>:/marche/server.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="rescanBtn">
           <property name="text">
            <string>Rescan</string>
           </property>
           <property name="icon">
            <iconset resource="../res/marche-gui.qrc">
             <normaloff>:/marche/server-cast.png</normaloff>:/marche/server-cast.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="reloadBtn">
           <property name="text">
            <string>Reload jobs</string>
           </property>
           <property name="icon">
            <iconset resource="../res/marche-gui.qrc">
             <normaloff>:/marche/arrow-circle-225.png</normaloff>:/marche/arrow-circle-225.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="cachePanel" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>180</width>
              <height>19</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblCached">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);</string>
            </property>
            <property name="text">
             <string>Cached credentials:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblCachedUserCreds">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);</string>
            </property>
            <property name="text">
             <string>{username}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="clearCredBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Clear cached credentials</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="hostList">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="surface" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAdd_host">
   <property name="icon">
    <iconset resource="../res/marche-gui.qrc">
     <normaloff>:/marche/server.png</normaloff>:/marche/server.png</iconset>
   </property>
   <property name="text">
    <string>Add host</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../res/marche-gui.qrc">
     <normaloff>:/marche/door-open.png</normaloff>:/marche/door-open.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res/marche-gui.qrc">
     <normaloff>:/marche/logo-new.svg</normaloff>:/marche/logo-new.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="../res/marche-gui.qrc">
     <normaloff>:/marche/logo-new.svg</normaloff>:/marche/logo-new.svg</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionScan_network">
   <property name="icon">
    <iconset resource="../res/marche-gui.qrc">
     <normaloff>:/marche/server-cast.png</normaloff>:/marche/server-cast.png</iconset>
   </property>
   <property name="text">
    <string>Scan network</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="../res/marche-gui.qrc">
     <normaloff>:/marche/arrow-circle-225.png</normaloff>:/marche/arrow-circle-225.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLoad_session">
   <property name="icon">
    <iconset resource="../res/marche-gui.qrc">
     <normaloff>:/marche/folder-open.png</normaloff>:/marche/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>Load session</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_session_as">
   <property name="icon">
    <iconset resource="../res/marche-gui.qrc">
     <normaloff>:/marche/disk.png</normaloff>:/marche/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save session as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../res/marche-gui.qrc">
     <normaloff>:/marche/drill.png</normaloff>:/marche/drill.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/marche-gui.qrc"/>
 </resources>
 <connections/>
</ui>
